### PR TITLE
Add server-side init call

### PR DESCRIPTION
**Pull request by samuelWilliams99**
Just creates an init hook for server side, to match client side. (hook name `CFC_Notifications_init`)

### CODE REVIEW

File: lua/cfc_notifications/shared/sh_base.lua [hunk 1]
```diff
@@ -32,6 +32,11 @@ if SERVER then
     hook.Add( "PlayerDisconnected", "CFC_NotificationUnready", function( ply )
         CFCNotifications.playersReady[ply] = true
     end )
+
+    -- Give chance for other addons to load
+    timer.Simple( 0, function()
```
Comment (sarahsturgeon) [inline lua/cfc_notifications/shared/sh_base.lua]: may as well make this 1 second

Comment (samuelWilliams99) [reply lua/cfc_notifications/shared/sh_base.lua]: fair

Comment (samuelWilliams99) [reply lua/cfc_notifications/shared/sh_base.lua]: Actually, client side initialisation runs on the Initialise hook, gonna use that instead

**Comment by samuelWilliams99** [comment]
@brandonsturgeon pls approve b0ss

File: lua/cfc_notifications/shared/sh_base.lua [hunk 2]
```diff
@@ -34,7 +34,7 @@ if SERVER then
     end )
 
     -- Give chance for other addons to load
-    timer.Simple( 0, function()
+    hook.Add( "Initialize", "cfc_notifications_init", function()
```
Comment (plally) [inline lua/cfc_notifications/shared/sh_base.lua:37]: your hook doesnt follow the cfc style guidelines 😲

Comment (samuelWilliams99) [reply lua/cfc_notifications/shared/sh_base.lua:37]: ah heck, this project was made before those existed lol

Comment (samuelWilliams99) [reply lua/cfc_notifications/shared/sh_base.lua:37]: it'll need a few changes, i can make an issue for it if you're ok with merging as it is rn?

Comment (sarahsturgeon) [reply lua/cfc_notifications/shared/sh_base.lua:37]: 🚓

**Review by sarahsturgeon — APPROVED** [review]

**Pull request merged**